src/algorithms/fardetectors/FarDetectorLinearProjection.h

```diff
@@ -17,18 +17,16 @@
 namespace eicrecon {
 
 using FarDetectorLinearProjectionAlgorithm =
-    algorithms::Algorithm<algorithms::Input<edm4eic::TrackSegmentCollection>,
+    algorithms::Algorithm<algorithms::Input<edm4eic::TrackCollection>,
                           algorithms::Output<edm4eic::TrackParametersCollection>>;
```

Comment:
(Just a thought)
For Acts tracking, we actually output projections as TrackSegments.